src/styles/components/advanced-section/styles.less

```diff
@@ -0,0 +1,16 @@
+& when (@advanced-section-enabled) {
+  .advanced-section {
+    &-content {
+      margin-top: @pod-margin-top;
+    }
+
+    &-label,
+    &-label:hover {
+      text-decoration: none;
+
+      .icon {
+        margin-right: @base-spacing-unit * 1/4;
+      }
+    }
+  }
+}
```

Comment:
Let's add the media queries to this and adjust the `.advanced-section-content` margin with the proper `@pod-margin-top-screen-*` and `@base-spacing-unit-screen-*` variables.